### PR TITLE
Modified docker script to mount tns and wallet_dir in case of assess-migration-bulk command's --fleet-config-file flag

### PR DESCRIPTION
**Pull request by sanyamsinghal**

### CODE REVIEW

**Comment by sanyamsinghal** [comment]
https://jenkins.dev.yugabyte.com/job/users/job/yb-voyager-testing/job/yb-voyager-testing-pipeline/3227/

**Comment by sanyamsinghal** [comment]
Final:
1. All except Oracle live - https://jenkins.dev.yugabyte.com/job/users/job/yb-voyager-testing/job/yb-voyager-testing-pipeline/3292/

File: docker/yb-voyager-docker [hunk 1]
```diff
@@ -99,6 +98,7 @@ map_host_path_inside_container() {
 			argv[${j}]=$(realpath "${argv[${j}]}")
 			dir_path=${argv[${j}]}
 		fi
+        echo "mapping host path ${dir_path} for flag ${flag_name}"
```
Comment (ShivanshGahlot) [inline docker/yb-voyager-docker:101]: We can remove this echo

File: docker/yb-voyager-docker [hunk 2]
```diff
@@ -116,87 +116,132 @@ map_host_path_inside_container() {
...
 
+# function to handle the mapping and storage of directory paths
+handle_directory_flag() {
+    local flag=$1
+    local path=$2
+    # Map the path to Docker volume
+    map_host_path_inside_container "$flag" "$path"
+    # Store the flag and path in metadata file if necessary
+    if [[ " ${flags_to_store[@]} " =~ " ${flag} " ]]; then
+        echo "${flag} ${path}" >> $command_metadata_file_name
```
Comment (ShivanshGahlot) [inline docker/yb-voyager-docker:127]: This one can be removed too

File: docker/yb-voyager-docker [hunk 3]
```diff
@@ -116,87 +116,132 @@ map_host_path_inside_container() {
...
+    else
+        tns_admin="${ORACLE_HOME}/network/admin"
+    fi
+    
+    if [ ! -d "$tns_admin" ]; then
+        echo "TNS_ADMIN directory does not exist: $tns_admin"
+        return 1  # Exit the function if TNS_ADMIN directory does not exist
+    fi
+
+    echo "current val of TNS_ADMIN: $tns_admin"
```
Comment (ShivanshGahlot) [inline docker/yb-voyager-docker:178]: We can probably remove this too

File: docker/yb-voyager-docker [hunk 4]
```diff
@@ -116,87 +116,132 @@ map_host_path_inside_container() {
...
+    exported_vars="${exported_vars} -e TNS_ADMIN=$tns_admin"
+}
+
+
+process_flag() {
+    local flag=$1
+    local index=$2
+    local retVal=0
+
+    echo "processing flag ${flag} value ${argv[$((index+1))]}"
```
Comment (ShivanshGahlot) [inline docker/yb-voyager-docker:204]: This can be removed

**Review by ShivanshGahlot — APPROVED** [review]
LGTM. Just the few echo statements that can be removed

**Pull request merged**